<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="text">
     <string>Designed by : mm1994uestc</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <italic>true</italic>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="text">
     <string>2D DrawPainter</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <italic>true</italic>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="text">
     <string>Qt interface Demo!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="TextInsert">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>361</width>
      <height>161</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="TextEdit">
     <attribute name="title">
      <string>TextEdit</string>
     </attribute>
     <widget class="QPlainTextEdit" name="InputContent">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>171</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Input Content:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="X_Pos">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="Y_Pos">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Content Position:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TextDraw">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Draw</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TextDelet">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Delet</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="DrawLine">
     <attribute name="title">
      <string>DrawLine</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="DrawFram">
     <attribute name="title">
      <string>DrawFrame</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Code">
     <attribute name="title">
      <string>Code</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>CODE-39</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CODE-128</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EAN-128</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CODEBAR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>UPC-A</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>65</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Content</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Code Type</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Pic">
     <attribute name="title">
      <string>Picture</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
